content/installation/netcore/usage/UsingDotNetAzure.md

```diff
@@ -0,0 +1,21 @@
+<!--
+title: "Using Contrast .NET on Microsoft Azure"
+description: "Using Contrast .NET on Microsoft Azure"
+tags: "configuration microsoft azure cloud agent installation .Net"
+-->
+
+Use the Contrast .NET agent to analyze ASP.NET applications running on Azure Virtual Machines (VMs), Cloud Services, Mobile Services or Azure App Service (formerly Azure Web Sites). Follow the instructions below to set up the .NET agent in your environment. 
```

Comment:
Have you guys decided when you're adding Azure content? If we're not planning on it for the first publication, I'll remove this article from the page structure.

Reply:
We will be supporting netcore on things like Azure Virtual Machines, but not the majority of Azure netcore deployments, e.g. Azure App Service.

@DanFiedler

Reply:
@ldemidov or @DanFiedler ?

Reply:
No Azure content for NetCore right now.